slides/intro-prubeh.md

```diff
@@ -39,7 +39,7 @@ Dej Chci poradit
 
 <!-- .slide: data-state="c-slide-task" -->
 
-# Hned si to vyzkoušíme
+# Hned si to <a href="https://czechitas.herokuapp.com/coach">vyzkoušíme</a>
```

Comment:
@iBobik adresa pro coache proč?

Reply:
Otevřel jsem to na projektoru, takže viděly jak to naskakuje. Takový motivační prvek, jakože to má smysl a my to hned uvidíme.

Reply:
Na projektoru ok. 

Jinak je to záměrně schovaný v jejich view, protože se chodily dívat, jestli maj ostatní hotovo. Což je nevím jestli je špatně nebo dobře vzhledem k cíli workshopu, ale protože lístečky jsou napsaný tak jak jsou, tak to dost bourá užití, proto to samy neviděj a vidět nemaj. To spíš jen na vysvětlenou.

Slajdy pak dostanou, ale to už je to jedno, takže 👍

Reply:
Naopak právě jsem měl pocit, že díky tomu ukázání na projektoru (jednorázovému) lépe chápou přínos. Ve spojení s tím, že jsem dal koučům úkol to taky sledovat, tak to skutečně aktivně využívaly.